core-bundle/src/DataContainer/ClipboardManager.php

```diff
@@ -113,6 +118,40 @@ public function clearAll(): void
         $session->set(self::SESSION_KEY, []);
     }
 
+    public function isCutMode(string $table): bool
```

Comment:
```suggestion
    public function isModeCut(string $table): bool
```

So it is consistent with the `MODE_CUT` constant. WDYT?

Reply:
To me that is not fluent to read. The constants are grouped by prefix, therefore the "incorrect" wording, but for method names I try to keep them readable.